package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "lit-kigali1-frontend",
+  "version": "1.0.0",
+  "description": "Authors Haven Front End",
+  "main": "index.js",
+  "scripts": {
+    "test": "jest --runInBand",
```

Comment:
@Nziranziza You can remove --runInBand so that we write tests that don't depend on each other